<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTableView" name="postTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>База данных:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Users</string>
    </property>
    <item>
     <property name="text">
      <string>Users</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Couriers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Employees</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PostOffices</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Countries</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cities</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reviews</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DepartureInformation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Letters</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Packages</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionCouriers">
   <property name="text">
    <string>Couriers</string>
   </property>
  </action>
  <action name="actionEmployees">
   <property name="text">
    <string>Employees</string>
   </property>
  </action>
  <action name="actionCountries">
   <property name="text">
    <string>Countries</string>
   </property>
  </action>
  <action name="actionCities">
   <property name="text">
    <string>Cities</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
